projman_filler/interop_run_stats_parser.py

```diff
@@ -131,7 +131,7 @@ def _get_conversion_results(self) -> list:
                     # These are the same for the lane across all reads
```

Comment:
Since this value is the same for all reads, we do not need to loop through all of them? We could just use the first read? I also think we do not necessarily need to look at only non-index reads. I think a lot of lines can be removed from this function.  

Could you also add a unit test to verify that Reads and Reads Pf are consistently the same across all reads within the same lane?